### PR TITLE
Refactoring part 9

### PR DESCRIPTION
**Pull request by IvanGoncharov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -840,6 +845,16 @@ function getValue(object) {
 }
 
 /*
+ * Strip string from leading/trailing space character, including space,
+ * tab, form feed, line feed, etc.
+ * @param string {string} - String to strip.
+ * @returns {string} - striped string
+*/
+function strip(string) {
+  return string.replace(/^\s*|\s*$/g, '');
```
Comment (mohsen1) [inline index.js]: How is it different than `String.prototype.trim`?

Comment (IvanGoncharov) [inline index.js]: Fixed
Good catch :+1: 
For some reason I thought `trim` strip only whitespaces and not tabs and newlines.
But actually it strip exactly same things that `\s` do.

**Comment by IvanGoncharov** [comment]
@mohsen1 Can you please review it.

**Comment by mohsen1** [comment]
Can you work on some tests on these? Like name of body parameters and global types.

**Comment by IvanGoncharov** [comment]
@mohsen1 I'm thinking about creating `test/input/fixable` schema specially to test very common errors in specs that fixed during conversion.
What do you think?

Also last npm release was more than two months ago.
IMHO, it could be `1.0.0` release. 
What is criterias for new release?

**Comment by mohsen1** [comment]
I'm fine with 1.0 release. I wish we had more tests around new functionality

**Comment by IvanGoncharov** [comment]
>  I wish we had more tests around new functionality

I agree with you.
Right now my workflow is to try to convert as much possible Swagger 1.x documents.
So after I change something I test it on my [pool](https://docs.google.com/spreadsheets/d/14zxKcW_Pzu5aYI3Tnwe5ph2Ru2pkSP8yHWEQhx3t8pI/edit#gid=75279491) of Swagger specs.
And if it fix some spec without breaking previously converted ones I make PR.
So my fixes always based on some real problem from Swagger specs that I found in the wild.

But at the same time it only me who have access to this "test suit".
And it's severely limiting other people ability, to test their changes.
Someday one of "clever" fixes could be broken without any regression.

It would be very helpful if you enable [coveralls.io integration](https://coveralls.zendesk.com/hc/en-us/articles/201769715-Javascript-Node).
Afterwards we can see current situation and work on improving test coverage.

**Comment by mohsen1** [comment]
You can add Istanbul for coverage which will run in TravisCI. Lets add those tests and release a 1.0.0

Thanks you!

**Pull request merged**